website/mantemplates/crowdin-distribution-edit.adoc

```diff
@@ -0,0 +1,18 @@
+:includedir: ../generated-picocli-docs
+:command: crowdin-distribution-edit
+
+== crowdin distribution edit
+
+include::{includedir}/{command}.adoc[tag=picocli-generated-man-section-description]
+
+include::{includedir}/{command}.adoc[tag=picocli-generated-man-section-synopsis]
+
+include::{includedir}/{command}.adoc[tag=picocli-generated-man-section-arguments]
+
+include::{includedir}/{command}.adoc[tag=picocli-generated-man-section-commands]
+
+include::{includedir}/{command}.adoc[tag=picocli-generated-man-section-options]
+
+include::{includedir}/{command}.adoc[tag=picocli-generated-man-section-footer]
+
+See the link:/crowdin-cli/commands/crowdin-bundle-list[`crowdin bundle list` command] to get the list of bundles including their hashes.
```

Comment:
This feels a bit incorrect. Did you mean the `distribution list` command instead of `bundle list`?